providers/dns/yandex/yandex.toml

```diff
@@ -1,4 +1,4 @@
-Name = "Yandex"
+Name = "Yandex PDD"
 Description = '''
 '''
 URL = "https://yandex.com/"
```

Comment:
Sorry for not catching it last time. I propose to change it to `https://yandex.com/dev/domain/doc/reference/dns-add.html` or `https://pdd.yandex.com`: the current URL is the wrong one.